src/main/java/com/xl88/cf401d4/lab/Songr/HelloWorldController.java

```diff
@@ -0,0 +1,38 @@
+package com.xl88.cf401d4.lab.Songr;
+
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Arrays;
+
+@RestController
+public class HelloWorldController {
+
+    @GetMapping("/hello")
+    public  String getHelloWorld(){
+        return "Hello, world";
+    }
+
+    @GetMapping("/capitalize/{sentence}")
+    public String getCapitalize(@PathVariable String sentence){
+        return sentence.toUpperCase();
+    }
+
+    @GetMapping("/reverse")
+    @ResponseBody
+    public String getReversedSentence(@RequestParam(name="sentence") String sentence){
+
+        String[] arr = sentence.split(" ");
+        String s="";
+        for(int i=0;i<arr.length/2;i++){
```

Comment:
A StringBuilder is more efficient when concatenating lots of strings

Reply:
okay. StringBuilder..